lte/gateway/python/scripts/subscriber_cli.py

```diff
@@ -182,6 +187,7 @@ def create_parser():
     subparsers = parser.add_subparsers(title="subcommands", dest="cmd")
     parser_add = subparsers.add_parser("add", help="Add a new subscriber")
     parser_del = subparsers.add_parser("delete", help="Delete a subscriber")
+    parser_purge = subparsers.add_parser("purge", help="Purge a subscriber")
```

Comment:
What is the difference between "delete subscriber" and "purge subscriber" ? Maybe it is explained in some doc and apologize for my ignorance.